src/lib/account_update.ts

```diff
@@ -245,6 +246,7 @@ let Permissions = {
     setTokenSymbol: Permission.signature(),
     incrementNonce: Permission.signature(),
     setVotingFor: Permission.signature(),
+    setTiming: Permission.signature(),
```

Comment:
These are the permissions that get applied by default when you deploy a zkApp. I think to prevent the "malicious zkApp developer" case that you outlined, the default for `setTiming` should be "proof". Note: we changed the default for `send` to "proof" as well on `main`, even though here it's still "signature"